default.config.yml

```diff
@@ -34,19 +35,28 @@ vagrant_cpus: 2
 # The web server software to use. Can be either 'apache' or 'nginx'.
 drupalvm_webserver: apache
 
-# Set this to false if you are using a different site deployment strategy and
-# would like to configure 'vagrant_synced_folders' and 'apache_vhosts' manually.
-build_makefile: true
+# Set this to 'true' and add a drupal.make.yml next to your config.yml if you are
+# using a drush makefile based site deployment strategy.
+build_makefile: false
 drush_makefile_path: "{{ config_dir }}/drupal.make.yml"
 
-# Set this to false if you don't need to install drupal (using the drupal_*
+# Set this to 'false' if you are using a different site deployment strategy and
+# would like to configure 'vagrant_synced_folders' and 'apache_vhosts' manually.
+# If there's a composer.json file present in 'drupal_composer_path', it will be
+# used instead of the 'drupal_composer_package'.
+build_composer: true
+drupal_composer_path: "/var/www/drupalvm/drupal"
+drupal_composer_package: "drupal-composer/drupal-project:8.x-dev"
```

Comment:
You might be correct, `drupal-project` adds some complexity. However, with this approach we don't need to maintain a `composer.json` at all, and it also removed some steps from the quick start guide.

But if you prefer, I can add a basic `composer.json` and maybe add the `create-project` command installation as an option. `build_composer_project: true` or something.